web/templates/settings/organization_permissions_admin.hbs

```diff
@@ -236,7 +236,7 @@
         <div id="org-msg-deletion" class="settings-subsection-parent">
             <div class="subsection-header">
                 <h3>{{t "Message deletion" }}
-                    {{> ../help_link_widget link="/help/edit-or-delete-a-message#delete-a-message" }}
+                    {{> ../help_link_widget link="/help/delete-a-message#delete-a-message-completely" }}
```

Comment:
I think this one can just go to the `help/delete-a-message` page. Before Alya's edits in commit 084331010c264076a345b6b0e07b50f33944c424, that's more or less the link went.

Reply:
Oh, I see. Hm, I was thinking we might want to link directly to the specific section that documents the [message deletion feature](https://zulip.com/help/edit-or-delete-a-message#delete-a-message-completely) to avoid potential confusion about this org permission. But I think it would also be okay if the link is just `help/delete-a-message`. What do you think?

Reply:
I guess I was thinking that the link to configure these settings is in the main description of that article, which you don't immediately see if the link goes to the **Delete a message completely** header.

Reply:
Yeah, that makes sense. Done!